### PR TITLE
test: e2e add test of clone and delete checks

### PR DESCRIPTION
**Pull request by karel-rehor**
* Adds a test to clone and delete checks 
* Waits in some existing tests improved with intercept
* Data-testids for check cards made more specific

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 62 | :yellow_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 93 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=3120)

File: cypress/e2e/shared/checks.test.ts [hunk 1]
```diff
@@ -545,4 +573,173 @@ describe('Checks', () => {
...
+        cy.getByTestID(`check-card ${deadmanCheck.name}`).should('be.visible')
+        cy.getByTestID('check-card--name').should('have.length', 1)
+      })
+    })
+
+    // TODO fill in actual test
+    it.skip('can clone a threshold check', () => {
+      // 1. create threshold over API
+    })
+  })
```
Comment (genehynson) [inline cypress/e2e/shared/checks.test.ts:744]: I'd rather just remove this blank test from the PR and add it when the test is ready of that's ok

**Review by genehynson — CHANGES_REQUESTED** [review]
Looks really good! Love the usages of cy.intercept and the addition of cy.getByTestIDHead. Just would like to remove that blank test and then this PR should be g2g

File: cypress/e2e/shared/checks.test.ts (hunk 1 above)
Comment (karel-rehor) [reply cypress/e2e/shared/checks.test.ts]: Removed

**Pull request merged**